azurerm/internal/services/resource/feature_resource_test.go

```diff
@@ -0,0 +1,111 @@
+package resource_test
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance/check"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/resource/parse"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/tf/pluginsdk"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+type FeatureResource struct {
+}
+
+func TestAccFeatureResource_basic(t *testing.T) {
+	if true {
+		t.Skip("Skipping due to acceptance test doesn't support having features toggled currently")
+	}
+	data := acceptance.BuildTestData(t, "azurerm_subscription_feature", "test")
+	r := FeatureResource{}
+
+	data.DataSourceTest(t, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+	})
+}
+
+func TestAccFeatureResource_requiresImport(t *testing.T) {
+	if true {
+		t.Skip("Skipping due to acceptance test doesn't support having features toggled currently")
+	}
```

Comment:
As above, this can be removed now the test env is ready in the alt subscription.